### PR TITLE
[sourcemap] Actually remove comments from TS output.

### PR DESCRIPTION
**Pull request by alloy**
The API wasn’t being used correctly previously. I didn’t see an obvious way to test this without adding unit tests to the test suite, which isn’t being done atm, let me know what you would prefer or if this is good enough.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "convert-source-map": "^1.3.0",
     "error-stack-parser": "^1.3.6",
     "eslint": "^3.8.1",
-    "lab": "^11.1.0",
+    "lab": "^13.1.0",
```
Comment (orta) [inline package.json]: A little bit arbitrary, bit that's OK with me

Comment (orta) [inline package.json]: I mean, tests pass 👍

Comment (alloy) [inline package.json]: How so? This version is the one that fixes the failure that was occurring on CI.

Comment (orta) [inline package.json]: Ah yeah, I didn't read the commit description - just came back to see it now :D

Comment (alloy) [inline package.json]: Ahhh, I see, yeah looking at the total diff would make it seem arbitrary indeed.

**Pull request merged**